std/neko/_std/String.hx

```diff
@@ -81,8 +88,11 @@ import haxe.iterators.StringKeyValueIterator;
 	public function lastIndexOf( str : String, ?startIndex : Int ) : Int {
 		untyped {
 			var last = -1;
+			var l = __dollar__ssize(this.__s);
 			if( startIndex == null )
-				startIndex = __dollar__ssize(this.__s);
+				startIndex = l;
+			if( __dollar__ssize(str.__s) == 0 )
+				return startIndex > l ? l : startIndex;
```

Comment:
Should this really return `l` or `l - 1`? Not sure what the spec is here.

Reply:
There is no spec for this yet. `"foobar".lastIndexOf("")` returns `6` on js.